chat-commands.js

```diff
@@ -1758,16 +1790,21 @@ exports.commands = {
 		}
 
 		let acAccount = (targetUser && targetUser.autoconfirmed !== userid && targetUser.autoconfirmed);
+		let displayMessage = '';
 		if (affected.length > 1) {
-			this.privateModCommand(`(${name}'s ` + (acAccount ? ` ac account: ${acAccount}, ` : "") + `locked alts: ${affected.slice(1).map(user => user.getLastName()).join(", ")})`);
+			displayMessage = `(${name}'s ` + (acAccount ? ` ac account: ${acAccount}, ` : "") + `locked alts: ${affected.slice(1).map(user => user.getLastName()).join(", ")})`;
+			this.privateModAction(displayMessage);
+			this.logModAction(displayMessage);
```

Comment:
Why are these separate?

Reply:
`privateModAction` should log the action by default. Use `sendModAction` to avoid logging.

Reply:
Oh, I see what's going on here. I'd leave it off, or log it separately inside the `modlog` function in some way.

`logModAction` should be removed completely.